Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pipeline {
     agent any
 
     tools {
-        maven 'apache-maven-latest'
+        maven 'apache-maven-3.8.6'
```

Comment:
is latest not >3.8.6 ?

Reply:
No, not yet, it is still 3.8.5 - see also https://wiki.eclipse.org/Jenkins#Apache_Maven